### ISSUE TITLE
Java: error: variable _ is already defined in enum OperatorEnum

### ISSUE THREAD

**Issue by cbrainerd** [Isilon/isilon_sdk#4]
## Issue

Generated Java language bindings fail to install using `mvn install` with:

> [ERROR] /Volumes/Git/swagger-codegen/isi_sdk_java_7_2/src/main/java/io/swagger/client/model/SyncJobPolicyFileMatchingPatternOrCriteriaItemAndCriteriaItem.java:[26,4] error: variable _ is already defined in enum OperatorEnum
> [ERROR] /Volumes/Git/swagger-codegen/isi_sdk_java_7_2/src/main/java/io/swagger/client/model/SyncJobPolicyFileMatchingPatternOrCriteriaItemAndCriteriaItem.java:[27,4] error: variable _ is already defined in enum OperatorEnum

and so on
## Analysis

The root of the problem appears to sanitation of enum names that occurs in [Java code gen](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java#L706) where characters outside of [A-Za-z0-9] are replaced with `_`. The operator enum is defined in the PAPI schema:

```
/4/sync/jobs?describe: 
                              "operator": {
                                "enum": [
                                  "==", 
                                  "!=", 
                                  ">", 
                                  ">=", 
                                  "<", 
                                  "<=", 
                                  "!"
                                ], 
```

Each of those values is converted to varable name `_` during Java codegen. 
## Steps to reproduce:

Generate swagger spec file:
`components/create_swagger_config.py -i <IP for 7.2.1 cluster> -u root -p a -e excluded_end_points_7_2.json -o output_7_2.json`

Generate Java language bindings:
`java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -i output_7_2.json -l java -o isi_sdk_java_7_2`

Attempt to install the generated bindings:
`cd isi_sdk_java_7_2`
`mvn install`

Among some other errors you will see:

> [ERROR] /Volumes/Git/swagger-codegen/isi_sdk_java_7_2/src/main/java/io/swagger/client/model/SyncJobPolicyFileMatchingPatternOrCriteriaItemAndCriteriaItem.java:[26,4] error: variable _ is already defined in enum OperatorEnum

**Comment by cbrainerd** [comment]
The obvious way to fix this would be to augment the enum variable name sanitation in AbstractJavaCodegen.java:706 with something like:

```java
        // string
        String var = value;
        var = var.replaceAll(">", "_GT_");
        var = var.replaceAll("<", "_LT_");
        var = var.replaceAll("=", "_EQ_");
        var = var.replaceAll("!", "_EX_");
        var = var.replaceAll("\\W+", "_").replaceAll("_+", "_").toUpperCase();
        if (var.matches("\\d.*")) {
            return "_" + var;
        } else {
            return var;
        }
```

**Comment by Atomicology** [comment]
Resolved by PR #57. SDK language bindings in Java were successfully generated for OneFS 8.1.0. I have a personal repository with the source and an updated example in the README for those who have interest: https://github.com/atomicology/isilon_sdk_java
